adamsandoval/adamsandoval.github.io
# Initial Slide animations

- This PR adds the initial functionality/animations which sync up menu items to their respective slides.
- Uses the [GSAP](https://greensock.com/get-started-js) library for animations due to it's popularity and performance. 

## To Verify

1. Click menu on the menu and hover over menu item
1. Slide connected with that menu item should animate in
1. Click on the menu item and the menu should fade away and the content for that slide should be displayed
1. Click the `Learn More` button for that slide and the content should animate out
1. If the slide is part of another page it'll reload the page to that url, otherwise it'll scroll down to the page title
